### ISSUE TITLE
Update model of BMI associated risks

### ISSUE THREAD

**Issue by HSalat** [alan-turing-institute/uatk-aspics#33]
- Change BMI array from 12 parameters to 16, see [fitData.csv](https://github.com/alan-turing-institute/uatk-aspics/files/9507783/fitData.csv) attached.
- Update every calculation of `oddBMI` from

```
oddBMI = params->bmi_multipliers[originNew * 3] + params->bmi_multipliers[originNew * 3 + 1] * [!!! BMI VARIABLE !!!] + params->bmi_multipliers[originNew * 3 + 2] * pown([!!! BMI VARIABLE !!!],2);
```

to 

```
oddBMI = params->bmi_multipliers[originNew * 4] + params->bmi_multipliers[originNew * 4 + 1] * [!!! BMI VARIABLE !!!] + params->bmi_multipliers[originNew * 4 + 2] * pown([!!! BMI VARIABLE !!!],2) + params->bmi_multipliers[originNew * 4 + 3] * pown([!!! BMI VARIABLE !!!],3);
```

replacing `[!!! BMI VARIABLE !!!]` accordingly

- update scenarios to `BMI = 26.7` / `if (new_bmi > 26.7)` / `if (new_bmi >= 27.7)`

**Comment by mfbenitezp** [comment]
lets define `!!! BMI VARIABLE !!!` appropriately., that would be the dynamic new_bmi, am I right?

**Comment by mfbenitezp** [comment]
So no need for scenario where `new_bmi = 40`?, no need for a scenario where `else if (new_bmi > 29.0)`

**Comment by HSalat** [comment]
> lets define !!! BMI VARIABLE !!! appropriately., that would be the dynamic new_bmi, am I right?

what is currently inside the brackets, which is different every time

**Comment by HSalat** [comment]
> So no need for scenario where new_bmi = 40?, no need for a scenario where else if (new_bmi > 29.0)

No, you replace the ones we modified the other day because the BMI model was skewed to new values without deleting anything

**Comment by mfbenitezp** [comment]
@HSalat would you mind take a quick look at this branch, at the moment I got 6 scenarios, including the one with BMI of 50., but also validate if the new equations are correct, so I get the green flag to run all of them. 

https://github.com/alan-turing-institute/uatk-aspics/blob/82186a87f8e13c7d4189ecf5de18ee81708205dc/aspics/kernels/ramp_ua.cl#L198

**Comment by HSalat** [comment]
I believe it is, except the BMI > 50 condition is not a scenario. It needs to be there all the time (so at the begining and no else), this is because the equations can curve down above that value (you might even end up with a negative odd for extreme value), and that part of the model is not supported by the data anyway.

**Comment by mfbenitezp** [comment]
Gotcha, on my way to the run the scenarios, fingers crossed

**Comment by mfbenitezp** [comment]
The outcomes still don't make sense. Ill keep checking tonight and Mon morning. Here is the latest branch in case you have spare time to double check https://github.com/alan-turing-institute/uatk-aspics/blob/939a676e82a050dc321854753e1b39da17fb02db/aspics/kernels/ramp_ua.cl#L198

**Comment by HSalat** [comment]
You need to move the BMI > 50 condition before the scenarios, otherwise it gets ignored
